pilot/pkg/xds/eds.go

```diff
@@ -183,7 +186,7 @@ func (s *DiscoveryServer) edsCacheUpdate(shard model.ShardKey, hostname string,
 		Namespace: namespace,
 	}: {}})
 	ep.mutex.Unlock()
-	if features.SendUnhealthyEndpoints && pushType == IncrementalPush {
```

Comment:
This is the actual change. Rest all formating changes

Reply:
Do we have any test coverage?

Reply:
We have test for this function - probably not for this specific case. Let me take a look

Reply:
@howardjohn added test. PTAL
@hzxuzhonghu  We can not set this feature to false by default because the new destination rule API depends on this. If users are sure that they do not use that api, they can set it to false.